base/ubi9-python-3.11/Pipfile

```diff
@@ -7,8 +7,8 @@ verify_ssl = true
 
 [packages]
 # Base packages
-wheel = "~=0.43.0"
-setuptools = "~=69.2.0"
+wheel = "~=0.44.0"
+setuptools = "~=73.0.0"
```

Comment:
meanwhile, 74.0.0 came out ;P I guess let's ignore that and stay with 73

https://pypi.org/project/setuptools/